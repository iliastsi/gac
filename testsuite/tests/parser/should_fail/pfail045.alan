hello() : proc
{
  if(true == true)
    return;
}
