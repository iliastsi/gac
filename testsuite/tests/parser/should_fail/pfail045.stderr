pfail045.alan:3:11: Error: Parse error on input `=='
	Cannot use comparison symbols in conditions

